os/fs/driver/mtd/smart.c

```diff
@@ -5388,7 +5385,7 @@ static int smart_journal_scan(FAR struct smart_struct_s *dev, bool print_dump)
 
 			/* If another case means that it is finished or power off during writing journal */
 			if (CHECK_JOURNAL_CHECKIN(log.status) && !CHECK_JOURNAL_CHECKOUT(log.status)) {
-				fdbg("none checkout data found... seq : %d\n", dev->journal_seq);
+				fdbg("Active Journal data found... seq : %d\n", dev->journal_seq);
```

Comment:
I think this is information message. If it is, please change fvdbg.

Reply:
It is kind of information but very important, We'd better show active journal (logically only one journal can be survived) .
So I will keep current style.